### PR TITLE
BigData Boutique guest blog post

### PR DESCRIPTION
**Pull request by spara**

### CODE REVIEW

**Comment by cnunciato** [comment]
> Proposed changes
> Unreleased product version (optional)
> Related issues (optional)

You can just delete this whole bit in future PRs -- it's not necessary.

File: content/blog/bigdata-boutique-guest-post/index.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+## Elasticsearch Sizing Process
+
+Our process begins with a sample data request. We ask the customer for their data (or a large sample of it), existing index mappings, expected queries, and finally known KPIs and SLAs. Our goal in the initial discovery phase is to reduce the number of free variables as much as possible because the more such variables, the more permutations we will have to test. Ideally, we want very few dimensions of freedom—usually in terms of hardware and configurations—so the number of permutations to test is on the order of dozens and not thousands.
+
+Once the initial framework has been decided, we run fully automated benchmarks to test some of the permutations our experience tells us are the most promising to deliver good results. The benchmarking process is smart enough to omit long tales and outliers, so the results are not going to lie as to the true nature of the configuration that was selected and tested.
```
Comment (cnunciato) [inline content/blog/bigdata-boutique-guest-post/index.md]: Original has this also, but happened to notice it:

```suggestion
Once the initial framework has been decided, we run fully automated benchmarks to test some of the permutations our experience tells us are the most promising to deliver good results. The benchmarking process is smart enough to omit long tails and outliers, so the results are not going to lie as to the true nature of the configuration that was selected and tested.
```

**Comment by cnunciato** [comment]
We should also remove `meta.png`. (FWIW, the intent of that file is to be replaced.)

File: content/blog/bigdata-boutique-guest-post/index.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+---
+title: "How Pulumi Drives Our Elasticsearch Capacity and Cost Optimization Service"
+date: 2020-03-05
+meta_desc: "BigData Boutique uses Pulumi to determine optimal Elasticsearch configurations for their customers"
+meta_image: bdbq-pulumi.png
+authors:
+    - sophia-parafina
```
[the remark this answers is not in the document]
Comment (spara) [reply content/blog/bigdata-boutique-guest-post/index.md]: done

File: content/blog/bigdata-boutique-guest-post/index.md (hunk 1 above)
Comment (spara) [reply content/blog/bigdata-boutique-guest-post/index.md]: fixed

File: content/blog/bigdata-boutique-guest-post/index.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+Whether our customer is running on-prem or on a cloud, our [Elasticsearch Capacity Planning Service](https://bigdataboutique.com/services/elasticsearch/capacity-planning) exists for one purpose: to find the hardware solution with the optimum balance between cost and performance, and do it scientifically, so the answers are as accurate and as precise as possible.
+
+## Elasticsearch Sizing Process
+
+Our process begins with a sample data request. We ask the customer for their data (or a large sample of it), existing index mappings, expected queries, and finally known KPIs and SLAs. Our goal in the initial discovery phase is to reduce the number of free variables as much as possible because the more such variables, the more permutations we will have to test. Ideally, we want very few dimensions of freedom—usually in terms of hardware and configurations—so the number of permutations to test is on the order of dozens and not thousands.
```
[the remark this answers is not in the document]
Comment (spara) [reply content/blog/bigdata-boutique-guest-post/index.md]: fixed

File: content/blog/bigdata-boutique-guest-post/index.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+- - -
+
+*Capacity Planning and Cost Optimization of Elasticsearch clusters requires a special level of expertise and automation. Here is how we use Pulumi to launch long-running benchmarks to correctly identify the right configuration for our customers’ Big Data clusters.*
+
+<table style="width:100%" cellspacing="0" cellpadding="0">
+  <tr>
+    <td>{{< figure src="portrait.PNG" alt="Itamar Sy-Hershko" >}}</td>
+    <td>At <a href="https://bigdataboutique.com/">BigData Boutique</a>, we are continually challenged by our customers—whether it’s complex Big Data challenges we are asked to solve, reducing costs for sophisticated systems holding mountains of data, improving performance for high-demand systems, or challenges faced in building a data-driven system.</td>
+  </tr>
+</table>
```
[the remark this answers is not in the document]
Comment (spara) [reply content/blog/bigdata-boutique-guest-post/index.md]: removed, added Itamar as a guest

File: content/blog/bigdata-boutique-guest-post/index.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    - Elasticsearch
+    - cost optimization
+---
+
+**Guest Article:** [Itamar Syn-Hershko](https://twitter.com/synhershko), Founder and CTO of [BigData Boutique](https://bigdataboutique.com/) shows how they use Pulumi to benchmark Elasticsearch configurations across cloud providers. Pulumi enables BigData Boutique to test deployments in parallel and gather metrics to produce performant and cost-effective solutions for its customers. The original article was published on their [company blog](https://blog.bigdataboutique.com/2020/03/how-pulumi-drives-our-elasticsearch-capacity-planning-and-cost-optimization-service-jx8qlu).
+<br>
+<br>
+<!--more-->
+
+- - -
```
[the remark this answers is not in the document]
Comment (spara) [reply content/blog/bigdata-boutique-guest-post/index.md]: added

**Pull request merged**